### ISSUE TITLE
Deploy Group Template Type Error

### ISSUE THREAD

**Issue by RedRhombus** [KelvinTegelaar/CIPP#1435]
### Description

1. IDENTITY MANAGEMENT
2. Administration
3. Deploy Group Templates
4. Select a Tenant > Next
5. Select a group to deploy > Next
6. Error is on this screen. 
 
Occurs for all group types and all tenants. Have tried creating new groups as well as importing from other tenants. 

![image](https://user-images.githubusercontent.com/39447115/227100171-2114cad8-7c6f-48a1-88ce-a5fb20f47744.png)

### Environment data

```PowerShell
Non-sponsored instance 
Front end version number: 3.2.1
Back end version number: 3.2.3
Tried Tenant Cache Clear: true 
Tried Token Cache Clear: true
```

**Comment by RedRhombus001** [comment]
> ### Description
> 1. IDENTITY MANAGEMENT
> 2. Administration
> 3. Deploy Group Templates
> 4. Select a Tenant > Next
> 5. Select a group to deploy > Next
> 6. Error is on this screen.
> 
> Occurs for all group types and all tenants. Have tried creating new groups as well as importing from other tenants.
> 
> ![image](https://user-images.githubusercontent.com/39447115/227100171-2114cad8-7c6f-48a1-88ce-a5fb20f47744.png)
> 
> ### Environment data
> ```powershell
> Non-sponsored instance 
> Front end version number: 3.2.1
> Back end version number: 3.2.3
> Tried Tenant Cache Clear: true 
> Tried Token Cache Clear: true
> ```

![image](https://user-images.githubusercontent.com/92956722/227104534-e76c4fe3-9ddb-49d2-85b4-8cb39016be7d.png)

**Comment by RedRhombus001** [comment]
When deploying one template this is the error:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/92956722/227104643-30663127-140b-45c0-b06d-daaeeccd78e6.png">

Dynamic query which works when creating manually.

user.assignedPlans -any (assignedPlan.servicePlanId -eq "c1ec4a95-1f05-45b3-a911-aa3fa01094f5" -and assignedPlan.capabilityStatus -eq "Enabled")

**Comment by BNWEIN** [comment]
This has been fixed in PR #292 on CIPP-API

https://github.com/KelvinTegelaar/CIPP-API/pull/292

**Comment by KelvinTegelaar** [comment]
fixed in dev
